docs/rules/Moq1201.md

```
@@ -1,10 +1,10 @@
 # Moq1201: Setup of async methods should use `.ReturnsAsync` instance instead of `.Result`
 
-| Item | Value |
-| --- | --- |
-| Enabled | True |
+| Item     | Value |
+| -------- | ----- |
+| Enabled  | True  |
 | Severity | Error |
-| CodeFix | False |
+| CodeFix  | False |
```

Markdownlint

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)